src/modules/utils/panel/screens/MainMenuScreen.cpp

```diff
@@ -82,7 +82,7 @@ void MainMenuScreen::setupConfigureScreen()
     mvs->addMenuItem("Z Home Ofs",
         []() -> float { void *rd; PublicData::get_value( endstops_checksum, home_offset_checksum, &rd ); return rd==nullptr ? 0.0F : ((float*)rd)[2]; },
         [this](float v) { send_gcode("M206", 'Z', v); },
-        0.1F
+        0.01F
```

Comment:
unrelated change. should be a separate pull request (and would probably not be accepted)